apps/io-func-wallet-solution-backend/src/app/config.ts

```diff
@@ -0,0 +1,58 @@
+import * as t from "io-ts";
+
+import { pipe } from "fp-ts/lib/function";
+import * as RE from "fp-ts/lib/ReaderEither";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { validate } from "../validation";
+import { FederationEntityMetadata } from "../entity-configuration";
+import {
+  CryptoConfiguration,
+  getCryptoConfigFromEnvironment,
+} from "../infra/crypto/config";
+import { readFromEnvironment } from "../infra/env";
+
+export const Config = t.type({
+  pagopa: t.type({
+    federationEntity: FederationEntityMetadata,
+    crypto: CryptoConfiguration,
+  }),
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Should we prefer a more flat structure? Why the need for `pagopa`?

```suggestion
  federationEntity: FederationEntityMetadata,
  crypto: CryptoConfiguration,
```

Reply:
yes it is usually useful when there are configurations for different entities like azure/aws etc. In our case I think we can omit it